src/test/java/service/impl/TransactionHandlerImplTest.java

```diff
@@ -0,0 +1,101 @@
+package service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import dao.FruitDaoImpl;
+import dao.FruitsDao;
+import db.Storage;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import service.TransactionHandler;
+import strategy.OperationStrategy;
+import strategy.OperationStrategyImpl;
+import strategy.handler.BalanceOperationHandler;
+import strategy.handler.OperationHandler;
+import strategy.handler.PurchaseOperationHandler;
+import strategy.handler.ReturnOperationHandler;
+import strategy.handler.SupplyOperationHandler;
+
+public class TransactionHandlerImplTest {
```

Comment:
you should also write tests for each OperationHandler

BalanceOperationHandler, SupplyOperationHandler and so on